docs/develop/auto_sdk/auto-xdm.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Auto XDM
+sidebar_position: 4
+description: Functions to utilize Auto XDM (Cross-Domain Messaging)
+keywords:
+    - Developers Documentation
+    - DevDocs
+    - SDK
+    - Auto SDK
+    - Auto XDM
+---
+
+### Cross-Domain Transfers
+
+- `transfer(api: ApiPromise, destination: ChainOrDomain, receiver: string, amount: Amount): Promise<SubmittableExtrinsic>`: Creates a transaction to transfer a specified amount to a receiver on a given destination, which can be either the consensus chain or a specific domain.
+
+#### Types:
```

Comment:
The "Types" sizing looks abnormally small here. Perhaps ### would be a better header size.